### PR TITLE
Andr1976 misc

### PR DESCRIPTION
**Pull request by andr1976**
Hi Daniel, a rather large chunk. We can wait till after 6.4 release, if you are busy. Anyway, a more rigorous valve sizing based on the full IEC/ANSI/ISA standard for both choked/non-choked conditions and Masoneilan two-phase, fully implemented for steady-state sizing. Gas, liquid sizing validated against Unisim and in-house calculation. Two-phase give a littler smaller values than Unisim (Which I suspect are using the older Fischer two-phase correction factor). The user can now also estimate Kvmax when the valve is specified for oulet pressure or dP. 
In dynamics I have kept the more simple formulas, I believe they are adequate for the level detail needed for dyanmic simulations, however, the multiphase model is also implemented both for P1, P2 and Wi as output. The P1 estimate is a not very beutiful Newton-like damped solver (not my proudest moment).

### CODE REVIEW

File: DWSIM.UI.Desktop.Editors/UnitOperations/General.cs [hunk 1]
```diff
@@ -917,15 +917,18 @@ void Initialize()
                         case Valve.CalculationMode.OutletPressure:
                             pos5 = 0;
                             break;
-                        case Valve.CalculationMode.Kv_Liquid:
+                        case Valve.CalculationMode.Kv_General:
```
Comment (DanWBR) [inline DWSIM.UI.Desktop.Editors/UnitOperations/General.cs:920]: please put Kv_General in the last enum position so it won't break existing simulations with Kv_liquid selected.

File: DWSIM.UI.Desktop.Editors/UnitOperations/General.cs [hunk 2]
```diff
@@ -938,12 +941,9 @@ void Initialize()
                                 valve.CalcMode = Valve.CalculationMode.DeltaP;
                                 break;
                             case 2:
-                                valve.CalcMode = Valve.CalculationMode.Kv_Liquid;
```
Comment (DanWBR) [inline DWSIM.UI.Desktop.Editors/UnitOperations/General.cs]: please put Kv_General in the last enum position so it won't break existing simulations with Kv_liquid selected.

File: DWSIM.UnitOperations/Unit Operations/Valve.vb [hunk 3]
```diff
@@ -48,16 +48,29 @@ Namespace UnitOperations
...
         Public Enum CalculationMode
             DeltaP = 0
             OutletPressure = 1
-            Kv_Liquid = 2
-            Kv_Gas = 3
-            Kv_Steam = 4
+            Kv_General = 2
+            Kv_Steam = 3
+            Kv_Liquid = 4
+            Kv_Gas = 5
```
Comment (DanWBR) [inline DWSIM.UnitOperations/Unit Operations/Valve.vb]: please put Kv_General in the last enum position so it won't break existing simulations with Kv_liquid selected. 

Do it like this:

Kv liquid = 2
Kv gas = 3
Kv steam = 4
Kv general = 5

**Review by DanWBR — CHANGES_REQUESTED** [review]

**Comment by andr1976** [comment]
Hi Daniel I have made the requested changes, also in the classic GUI.

**Review by DanWBR — APPROVED** [review]

**Pull request merged**